src/tex/appendix.tex

```diff
@@ -447,6 +447,18 @@ \subsection{Impact of sampling temperature}
 \end{figure}
 
 
+\clearpage
+\subsection{Summary of trends and recommendations}
+
+\oone model consistently leads across both \ChemBench corpus and \chembenchmini.  \ClaudeThreeFiveSonnet, \GPTFourO, \LlamaThreeOneFourZeroFiveBInstruct form the next tier of strong performers. These models show robust performance across all skills and difficulty levels. 
+
+Larger models generally perform better (\LlamaThreeOneFourZeroFiveBInstruct > \LlamaThreeOneSeventyBInstruct > \LlamaThreeOneEightBInstruct). However, smaller but well-designed models can compete with larger models. For example, \GemmaTwoNineBIt has an overall accuracy of 0.47 on \ChemBench corpus, which is only 18\% less accurate than the larger \LlamaThreeOneEightBInstruct model.
```

Comment:
```suggestion
Larger models generally perform better (\LlamaThreeOneFourZeroFiveBInstruct > \LlamaThreeOneSeventyBInstruct > \LlamaThreeOneEightBInstruct). However, some smaller models can compete with larger models. For example, \GemmaTwoNineBIt has an overall accuracy of 0.47 on the \ChemBench corpus, which is only 18\% less accurate than the largest model from the LLama-3.1 herd (\LlamaThreeOneEightBInstruct).
```

Reply:
I dont get this? Gemma is 9B Llama 3.1 model mentioned is 8B?